envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/assertions/ResponseAssertions.kt

```diff
@@ -9,6 +9,11 @@ fun ObjectAssert<Response>.isOk(): ObjectAssert<Response> {
     return this
 }
 
+fun ObjectAssert<Response>.isForbidden(): ObjectAssert<Response> {
+    matches { it.code() == 403 }
```

Comment:
Don't we need `body()?.close` ? In `pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt:414` we have it and a couple of lines below there is one as well. I think we might be leaking connections here.

Reply:
Changed to approve just make sure it works here @wookieJ @MarcinFalkowski

Reply:
good point, fixed